tests/accessgroups_test.go

```diff
@@ -0,0 +1,92 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	services "github.com/jfrog/jfrog-client-go/access/services/v2"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccessGroups(t *testing.T) {
+	initAccessTest(t)
+	t.Run("create", testCreateAccessGroup)
+	t.Run("update", testUpdateAccessGroup)
+	t.Run("delete", testDeleteAccessGroup)
+}
+
+func getTestAccessGroupParams() services.GroupParams {
+	group := services.GroupDetails{
+		Name:            fmt.Sprintf("test-%s", getRunId()),
+		Description:     "hello",
+		AutoJoin:        &falseValue,
+		AdminPrivileges: &trueValue,
+		Realm:           "internal",
+		RealmAttributes: "",
+		ExternalId:      "",
+	}
+	return services.GroupParams{GroupDetails: group}
+}
+
+func testCreateAccessGroup(t *testing.T) {
+	groupParams := getTestAccessGroupParams()
+	err := testAccessGroupService.Create(groupParams)
+	defer deleteAccessGroupAndAssert(t, groupParams.GroupDetails.Name)
+	assert.NoError(t, err)
```

Comment:
This error assertion should be placed right after the line that returns the error (before the defer).
The same goes for the `testUpdateAccessGroup` function.